core/controllers/admin.py

```diff
@@ -784,66 +784,72 @@ def get(self):
         self.render_json(response)
 
 
-class AddContributionReviewerHandler(base.BaseHandler):
-    """Handles adding reviewer for contributor dashboard page."""
+class AddContributionRightsHandler(base.BaseHandler):
```

Comment:
(Maybe) Future work: Should we have one handler "ContributionRightsHandlre" which will have get/post/delete function to handle fetch/add/removal of rights?